tests/functional/adapter/store_failures/test_store_failures.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.store_failures.fixtures import (
+    seed_csv,
+    table_model,
+    table_profile_yml,
+)
+
+
+class TestStoreFailuresBase:
+    """
+    Testing store_failures functionality
+    """
+
+    @property
+    def schema(self):
+        return "default"
+
+    # everything that goes in the "seeds" directory
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "seed.csv": seed_csv,
+        }
+
+
+class TestStoreFailuresTable(TestStoreFailuresBase):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "store_failures_tests",
+            "models": {
+                "+materialized": "table",
+                # "+properties": {"transactional": True},
+            },
+            # "seeds": {
+            # "+properties": {"transactional": True},
+            # },
+            "tests": {
+                "+store_failures": True,
+                # "+properties": {"transactional": True},
+            },
+        }
+
+    # everything that goes in the "models" directory
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "table_model.sql": table_model,
+            "table_store_failures.yml": table_profile_yml,
+        }
+
+    def test_run_seed_test(self, project):
```

Comment:
I tried to run the tests after commenting out `TrinoRelation#_is_exactish_match` and the tests succeed.
What am I missing? The tests should showcase the existing limitation in the framework.

Reply:
let me check again in our places later

Reply:
ok we understood. With this commit, the problem we mentions will be fixed by default. But still with some situation, the issue remains. We fixed the test to reflect this. With this test when you comment out the fix in relation.py it should fail.
https://github.com/starburstdata/dbt-trino/commit/a1d2267a8251715febe27ae2595045099e8e8c98